mscore/inspector/inspectorTextLineBase.cpp

```diff
@@ -197,6 +202,76 @@ void InspectorTextLineBase::valueChanged(int idx)
             inspector->update();
       }
 
+//---------------------------------------------------------
+//   hasBeginTextClicked
+//---------------------------------------------------------
 
-} // namespace Ms
+void InspectorTextLineBase::hasBeginTextClicked(bool checked)
```

Comment:
Would be better to name it like **on**HasBeginTextClicked perhaps?

Reply:
Yeah it is clearer, but the way without `on` is what's mostly used in this kind of circumstances in MuseScore.